Gems/Atom/Tools/AtomToolsFramework/Code/Source/Document/AtomToolsDocumentMainWindow.cpp

```diff
@@ -325,45 +325,50 @@ namespace AtomToolsFramework
-                CreatePropertyFromSetting(
-                    "/O3DE/AtomToolsFramework/AtomToolsDocumentSystem/EnableAutomaticReload",
-                    "Enable Automatic Reload",
-                    "Automatically reload documents after external modifications",
-                    true),
-                CreatePropertyFromSetting(
-                    "/O3DE/AtomToolsFramework/AtomToolsDocumentSystem/EnableAutomaticReloadPrompts",
-                    "Enable Automatic Reload Prompts",
-                    "Confirm before automatically reloading modified documents",
-                    true),
-                CreatePropertyFromSetting(
-                    "/O3DE/AtomToolsFramework/AtomToolsDocumentSystem/AutoSaveEnabled",
-                    "Enable Auto Save",
-                    "Automatically save documents after they are modified",
-                    false),
-                CreatePropertyFromSetting(
-                    "/O3DE/AtomToolsFramework/AtomToolsDocumentSystem/AutoSaveInterval",
-                    "Auto Save Interval",
-                    "How often (in milliseconds) auto save occurs",
-                    aznumeric_cast<AZ::s64>(250)),
-            }));
-        return groups;
+            { CreateSettingsPropertyValue(
+                  "/O3DE/AtomToolsFramework/AtomToolsDocumentSystem/DisplayWarningMessageDialogs",
+                  "Display Warning Message Dialogs",
+                  "Display message boxes for warnings opening documents",
+                  true),
+              CreateSettingsPropertyValue(
+                  "/O3DE/AtomToolsFramework/AtomToolsDocumentSystem/DisplayErrorMessageDialogs",
+                  "Display Error Message Dialogs",
+                  "Display message boxes for errors opening documents",
+                  true),
+              CreateSettingsPropertyValue(
+                  "/O3DE/AtomToolsFramework/AtomToolsDocumentSystem/EnableAutomaticReload",
+                  "Enable Automatic Reload",
+                  "Automatically reload documents after external modifications",
+                  true),
+              CreateSettingsPropertyValue(
+                  "/O3DE/AtomToolsFramework/AtomToolsDocumentSystem/EnableAutomaticReloadPrompts",
+                  "Enable Automatic Reload Prompts",
+                  "Confirm before automatically reloading modified documents",
+                  true),
+              CreateSettingsPropertyValue(
+                  "/O3DE/AtomToolsFramework/AtomToolsDocumentSystem/AutoSaveEnabled",
+                  "Enable Auto Save",
+                  "Automatically save documents after they are modified",
+                  false),
+              CreateSettingsPropertyValue(
+                  "/O3DE/AtomToolsFramework/AtomToolsDocumentSystem/AutoSaveInterval",
+                  "Auto Save Interval",
+                  "How often (in milliseconds) auto save occurs",
+                  aznumeric_cast<AZ::s64>(250)) });
```

Comment:
If you want, I think `250ll` expresses the same thing

Reply:
s64 is required by the settings registry and defined differently on different platforms. Probably something that should be brought to the attention of core if there's a distinction between the types.